harness/determined/core/_context.py

```diff
@@ -152,14 +157,21 @@ def init(
         preempt_mode (``core.PreemptMode``, optional): Configure the calling pattern for the
             ``core_context.preempt.should_preempt()`` method.  See
             :class:`~determined.core.PreemptMode` for more detail.  Defaults to ``WorkersAskChief``.
-        storage_manager: Internal use only.
+        checkpoint_storage (``str``, optional): A directory path specifying a shared filesystem
```

Comment:
> A directory path specifying a shared filesystem location or...

Just say "a directory to store checkpoint in" or something.  In local training mode, the "shared" adjective is confusing and not helpful.